docs/advanced/longhorn-v2.md

```diff
@@ -0,0 +1,85 @@
+
+Every node with an active Longhorn V2 Data Engine requires the following dedicated resources:
+
+- 1 CPU core for use by the Longhorn instance-manager pod
+- 2 GiB RAM (allocated as 1024 × 2 GiB huge pages)
+- At least one local NVMe disk for data storage
+
+## Limitations
+
+:::info important
+
+The Longhorn V2 Data Engine currently does not support the following operations:
+
+- Backing image creation and usage
+- Live migration
+- Volume cloning
+- Volume encryption
+- Volume expansion
+
+:::
+
+Within the Harvester context, volumes backed by the Longhorn V2 Data Engine must be added to virtual machines as extra disks. The boot disk of each virtual machine must still be added from an image that is backed by the Longhorn V1 Data Engine.
+
+Harvester is unable to live-migrate virtual machines with V2 volumes attached, which means those virtual machines will be shut down during Harvester cluster upgrades. Moreover, snapshots of V2 volumes cannot be created because snapshot and restoration functionality in Harvester relies on volume cloning.
+
+## Using the Longhorn V2 Data Engine
+
+The Longhorn V2 Data Engine is only available for newly created volumes. Existing volumes, virtual machine images and virtual machine root volumes will continue to use the V1 Data Engine.
+
+1. On the Harvester UI, go to **Advanced** > **Settings**.
+
+1. Set `longhorn-v2-data-engine-enabled` to `true`.
+
+  Harvester automatically loads the kernel modules required by the Longhorn V2 Data Engine, and attempts to allocate 1024 × 2 MiB-sized huge pages (for example, 2 GiB of RAM) on all nodes. 
+
+  Changing this setting automatically restarts RKE2 on all nodes but does not affect running virtual machine workloads.
+
+  :::tip
+
+  If you encounter error messages that include the phrase "not enough hugepages-2Mi capacity", allow some time for the error to be resolved. If the error persists, reboot the affected nodes.
+  
+  To disable the Longhorn V2 Data Engine on specific nodes (for example, nodes with less processing and memory resources), go to the **Hosts** screen and add the following label to the target nodes:
+
+    - label: `node.longhorn.io/disable-v2-data-engine`
+    - value: `true`
+
+  :::
+
+1. Go to the **Hosts** screen, and then add extra disks to each node as described in [Multi-disk Management](../host/host.md#multi-disk-management). 
+
+  Set the `Provisioner` of each extra disk to `Longhorn V2 (CSI)`.
+
+  :::note
+
+  Harvester sets the Longhorn disk driver to "auto" as described in [the Longhorn documentation](https://longhorn.io/docs/1.7.2/v2-data-engine/features/node-disk-support/). This means that NVMe disks will use the nvme bdev driver which provides the best performance. If testing with non-NVMe disks (e.g. SSDs), note that these will fall back to using the aio bdev driver, which requires that the disk size be an even multiple of 4096 bytes. Non-NVMe disks that do not meet this size constraint will not be added successfully.
+  :::
```

Comment:
```suggestion
  :::note

  Harvester sets the [Longhorn disk driver](https://longhorn.io/docs/1.7.2/v2-data-engine/features/node-disk-support/) to `auto` so that NVMe disks use the SPDK NVMe bdev driver, which provides the best performance.
  
  SSDs and other non-NVMe disks are managed using the SPDK AIO bdev driver, which requires a disk size that is an *even multiple of 4096 bytes*. Non-NVMe disks that do not meet this size requirement cannot be added.

  :::
```

Reply:
I added "SPDK" to the driver names because those seem to be official terms in the [SPDK doc](https://spdk.io/doc/bdev.html).